tests/_support/Config/Services.php

```diff
@@ -43,4 +44,20 @@ public static function uri(?string $uri = null, bool $getShared = true)
 
         return new URI($uri);
     }
+
+    /**
+     * The URI class provides a way to model and manipulate URIs.
+     *
+     * @param mixed $getShared
+     *
+     * @return stdClass
+     */
+    public static function someService($getShared = true)
```

Comment:
I think our convention is that a service name is all lower case.
This should be `someservice`.

Reply:
I did not know about that convention but there is nothing limiting the user not to use any letter casing.

Function names in PHP are case insensitive so it will work with any case.  And it make sense to me to use camel case for service names

Reply:
This must be lowercase.
https://github.com/codeigniter4/CodeIgniter4/blob/812af1e6285c352ae9b4da044fd7c1a0e8815d90/system/Config/Services.php#L180